companies/studydrive.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "studydrive",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "school"
+    ],
+    "name": "Studydrive GmbH",
+    "address": "Mehringdamm 32\n10961 Berlin\nDeutschland",
+    "phone": "+49 (030) 555 794 64",
```

Comment:
If I type that into the JSON generator, I (correctly) get the error that it doesn't match the format regex.

Reply:
WTF? This is different from what was in the input. Also, the format button in the JSON generator seems to be broken for me. I don't know the expected behavior, but it does definitely not format…

Reply:
As I said, there are some (a lot of) problems with the JSON forms thingy.

![Peek 2019-10-05 20-00](https://user-images.githubusercontent.com/4048809/66258906-25ff9f00-e79a-11e9-872d-63c39f8ff144.gif)

The format button (pressing enter in the input also works) works fine for me. The problem is that JSON forms doesn't seem to recognize that the input has changed. You need to manually change it, trigger a read and then change it back. Very annoying.

Similar issues occur when dragging and dropping text into the inputs for example. C&P-ing usually works, though.